src/lib/AnnotationControls.ts

```diff
@@ -160,5 +172,20 @@ export default class AnnotationControls {
         );
 
         regionButton.setAttribute('data-testid', 'bp-AnnotationsControls-regionBtn');
+        regionButton.onkeydown = (event: KeyboardEvent): void => {
+            if (!annotator) {
+                return;
+            }
+
+            if (event.key !== 'Escape') {
+                return;
+            }
+
+            this.resetControls();
+            annotator.toggleAnnotationMode(AnnotationMode.NONE);
```

Comment:
Can `annotator` be undefined here? Should the controls instead provide a callback to the parent, like `onReset`, which the parent can use to reset the mode?